lib/cocoapods/generate/configuration.rb

```diff
@@ -149,6 +149,9 @@ def ===(other)
 
       option :use_libraries, BOOLEAN, 'false', 'Whether to use libraries instead of frameworks', nil, nil, coerce_to_bool
 
+      option :generate_multiple_pod_projects, BOOLEAN, 'false', 'Whether to generate multiple Xcode projects', nil, nil, coerce_to_bool
```

Comment:
please add an integration test that sets both of these options to true?

Reply:
sure